pkgs/development/libraries/libbpkg/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib, stdenv
+, build2
+, fetchurl
+, libbutl
+
```

Comment:
```suggestion
```

Reply:
I like an empty line to separate packages from options, but I don't feel strongly about it.